public/_redirects

```diff
@@ -1,2 +1,9 @@
-/home301 / 301
-/home302 / 302
\ No newline at end of file
+/top / 301
+/snippets/top 301
+
+/api/* /:splat 301
+/concepts/* /:splat 301
+/getting-started/* /:splat 301
+/guides/* /:splat 301
+/helpers/* /:splat 301
+/middleware/* /:splat 301
```

Comment:
Testing this wrangler does not work. When I go to `/guides/jsx` it redirects me to `/jsx`

The following is working for me:
```
/api/* /docs/api/:splat 301
/concepts/* /docs/concepts/:splat 301
/getting-started/* /docs/getting-started/:splat 301
/guides/* /docs/guides/:splat 301
/helpers/* /docs/helpers/:splat 301
/middleware/* /docs/middleware/:splat 301
```

Reply:
Sorry for the delay, fixed in https://github.com/honojs/website/commit/194e9d8a89c439996ddcd5ea724959281dd041bf.